lib/config/rules/import.js

```diff
@@ -75,4 +75,6 @@ module.exports = {
   'import/max-dependencies': 'off',
   // Forbid unassigned imports
   'import/no-unassigned-import': 'off',
+  // Forbid anonymous values as default exports <FEEDBACK>
+  'import/no-anonymous-default-export': 'off',
```

Comment:
[Rule docs](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-anonymous-default-export.md)

Thoughts on this? I've seen our code export defaults before. Are there any specific types we should explicitly error?

Reply:
I think we can turn this on, we prefer named functions anyways, and exporting literals is a pretty uncommon case.